frameworks/bf/run-bf.js

```diff
@@ -0,0 +1,7 @@
+var execSync = require('child_process').execSync;
+module.exports.runBf = function(input) {
```

Comment:
Actually @kazk to be specific, it appears that I'm now getting a string of the format `<Buffer 48 65 6c 6c 6f 20 57 6f 72 6c 64 21>` (which is supposed to be "Hello World" in this example) when `runBF` is executed.  I've tried to use a RegExp replace on it, trying to treat the return value of `execSync` as a plain string but somehow the entire STDOUT becomes empty again instead.  I'm pretty sure that my RegExp is correct (after all, I'm just trying to remove the `<Buffer ` and `>` parts), but may you please help me check out this (commented out) part of my `runBF` function?

```javascript
return execSync('bf /workspace/solution.txt', {input})/* .replace(/^\<Buffer |\>$/g, "") */;
```

Or is my treatment of the return value of `execSync` as a plain string incorrect?